hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/impl/prefetch/TestBlockCache.java

```diff
@@ -67,7 +72,7 @@ public void testPutAndGet() throws Exception {
 
     assertEquals(0, cache.size());
     assertFalse(cache.containsBlock(0));
-    cache.put(0, buffer1);
+    cache.put(0, buffer1, CONF, new LocalDirAllocator(HADOOP_TMP_DIR));
```

Comment:
For Test* classes, using `BUFFER_DIR` is not helpful as they don't use `S3ATestUtils#prepareTestConfiguration`.

Hence, using `HADOOP_TMP_DIR` for Test* classes.